### PR TITLE
Replace ocean_wave version in levels

### PR DESCRIPTION
**Pull request by epeach**
Request from @mikeharv to replace the 'ocean_waves' image with a LtR 'flipped' version.

Steps not shown here:
* Upload new images to S3.
* Rebuild sprite costume library manifest for SpriteLab.
* Upload new manifest to S3.

The step captured in this PR is updating the version number of "ocean_waves" in the levels so they reference the new version.

This is candidate for getting deployed during restricted deployment.

Old:
Screenshot from 2021-11-22 16-12-49

New:
Screenshot from 2021-11-22 16-08-59

## PR Checklist:

- [x] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/config/scripts/levels/ErinPoembot.level [hunk 1]
```diff
@@ -0,0 +1,116 @@
+<GamelabJr>
```
Comment (maddiedierker) [inline dashboard/config/scripts/levels/ErinPoembot.level]: i don't think we want to commit this level, right?

**Review by maddiedierker — COMMENTED** [review]
i'm struggling to find the diff in the .level files -- can you elaborate on what the change is there?

File: dashboard/config/scripts/levels/ErinPoembot.level (hunk 1 above)
Comment (epeach) [reply dashboard/config/scripts/levels/ErinPoembot.level]: Yup! Thanks for catching.

**Comment by epeach** [comment]
I took a screenshot of the changed version of the file here. 
Screenshot from 2021-11-22 17-32-32

The changes between files should be localized just to the ocean_wave costume and within ocean_wave, it should only be the "version: <version>" and the "sourceUrl: /api/v1/animation-library/spritelab/<version>/category_video_games/ocean_wave.png", that changes. This is because the rest of the metadata has stayed the same, we've just uploaded a new version of this image that is flipped. Uploading this new flipped image created a new version number in S3, so I went through to all the old version numbers and updated them to use the new version number. Because this sprite is referenced in the animationJSONs for certain levels, I had to edit the level files (which unfortunately don't diff well).

**Review by maddiedierker — APPROVED** [review]
🌊 🌊 🌊

**Review by mikeharv — APPROVED** [review]
Looks great, thanks!

**Pull request merged**